### DISCUSSION TITLE
AMD 7700X

### DISCUSSION THREAD

**Discussion by jawjop** [isc30/ryzen-gpu-passthrough-proxmox#37] [Show and tell]
Just an FYI and a big thank you to the maintainer:

I can confirm this works for my 7700X. Have followed the readme instructions and generated the rom file. I can add it to the repo if the maintainer would like as well. Have tested this working in both a Linux and Windows guest. In my Linux guest, I was even able to pass through the GPU to a docker container and run GPU accelerated code on it via ROCM.

**Comment by isc30** [comment]
Hey that's great! Yes it would be nice to get your rom and vbios shared if you want :)

**Reply by jawjop** [reply]
No worries. I can't push to the repo (and subsequently create a PR with my branch) because it says I only have read access so I'll need access to do that (or if you want the files through another method, lmk)

**Reply by isc30** [reply]
Hey you should be able to fork the repo and open a PR with the changes

**Reply by jawjop** [reply]
Hey mate, apologies for the delay. I don't seem to have permission?

git push --set-upstream origin add-7700x-and-vbiosc
ERROR: Permission to isc30/ryzen-7000-series-proxmox.git denied to jawjop.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
